src/styles/style.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  margin: auto;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: $cellGap;
+  padding: $cellPadding;
+  max-width: $maxWidth;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $cellWidth;
+    height: $cellWidth;
+    border: 1px solid #000;
+    background-color: #eee;
+    transition: transform 0.5s, background-color 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffc0cb;
+      transform: translateY(-20px);
+    }
+  }
+
+  @for $day from 1 through 31 {
+    & :nth-child(#{$day})::before {
+      content: "#{$day}";
+    }
+  }
+
+  @each $value, $key in $startDay {
+    &--start--#{$value} :first-child {
+      margin-left: ($cellWidth * $key) + ($cellGap * $key);
+    }
+  }
+
+  @for $lastDay from 28 through 31 {
+    &--month-length--#{$lastDay} :nth-child(n + #{$lastDay}) {
+      display: none;
+    }
```

Comment:
to show 31 day I need to add to modifier 32 days etc
please fix this bug

![image](https://user-images.githubusercontent.com/57598825/113736221-937cea80-9705-11eb-8a6c-da7aaac03bf1.png)